### ISSUE TITLE
vis2.x: Binding for background-color does not work

### ISSUE THREAD

**Issue by ice987987** [ioBroker/ioBroker.vis-2#19]
**Describe the bug**
Binding for background-color does not work

**To Reproduce**

**Create a new view in vis**
Go to „Atribute“ -> section „CSS Hintergrund (background-...)
Enter a binding in: „-color:“ `{test:0_userdata.0.test;test == true ? '#ffff00' :: '#ff0000'}`

**Expected behavior**  
Binding is to be executed

**Screenshots & Logfiles**  
![image](https://user-images.githubusercontent.com/69053781/202529956-86a1db7c-2a1e-4a34-ace8-a16bd5c56c4e.png)

**Versions:**  
- Adapter version: ioBroker.vis v2.0.6
- JS-Controller version: v4.0.23
- Node version: v16.18.1
- Operating system: ioBroker.docker v7.2.1

**Comment by Areaman51** [comment]
Hello, 
is this problem solved? 

I want to change the backgroundcolour on night and day. If this work by binding it would be nice.

**Comment by Dickschieder** [comment]
I don't think so. I've Version 2.9.47 and get mostly the default colour in bindings on other widgets. I stopped creating bindings for the moment.

**Comment by dedeha1** [comment]
Have same problem. Its an important function in most of my views. So can not understand that an issue from 2022 is still open.

**Comment by Feuer-sturm** [comment]
@dedeha1 if this issue is important for you please raise a 👍  in the first post.

**Comment by oweitman** [comment]
workaround für vis2

auf jedem view der eingefärbt werden soll ein html widget legen und folgende
https://forum.iobroker.net/topic/76281/views-hintergrundfarbe-zentral-steuern/10
